android/src/main/java/com/rnfs/Uploader.java

```diff
@@ -117,13 +116,13 @@ private void upload(UploadParams params, UploadResult result) throws Exception {
             for (ReadableMap map : params.files) {
                 request.writeBytes(fileHeader[fileCount]);
                 File file = new File(map.getString("filepath"));
-                FileInputStream fis = new FileInputStream(file);
                 int fileLength = (int) file.length();
                 int bytes_read = 0;
                 int bytesReadTotal = 0;
-                int buffer_size = fileLength / 100;
+                int buffer_size =(int) Math.ceil(fileLength / 100.f);
```

Comment:
I think we should use a fixed buffer size depending on the available memory size of the device instead of 1 percent of the file length. If the user wants to upload a very big file, the app could crash because of "Out of memory" exception.
Since this file length based approach was already present before this PR, it has not to be done here. Just an idea for further discussions and for next steps :) 
What do you think?

Reply:
That's a big problem. 😱 
I will find a solution!
Thanks for your reminder!

Reply:
What do you think, shall we solve this buffer size issue with another PR in favour of a fast merge of this one, @hank121314 ?

Reply:
Could you give me one day to fix this issue?
I think I can resolve this quickly!

Reply:
Sure. Looking forward

Reply:
Hi! I use Runtime.freememory() to compare with fileLength /  100.
I think this might be solved OOM issue.